config/config.go

```diff
@@ -522,6 +522,8 @@ type TiKVClient struct {
 	CoprCache            CoprocessorCache `toml:"copr-cache" json:"copr-cache"`
 	// TTLRefreshedTxnSize controls whether a transaction should update its TTL or not.
 	TTLRefreshedTxnSize int64 `toml:"ttl-refreshed-txn-size" json:"ttl-refreshed-txn-size"`
+	// Ensure external consistency when async commit or 1PC is enabled.
+	ExternalConsistency bool `toml:"external-consistency" json:"external-consistency"`
```

Comment:
Consider adding this to config test and config example, and explain its pros and cons in comments of config example?

Reply:
OK. I will do.

Reply:
I think we better rename it to `StrictExternalConsistency`.